matrix-appservice-irc/pipeline.yml

```diff
@@ -34,3 +44,14 @@ steps:
       - docker#v3.0.1:
           image: "node:12"
           mount-buildkite-agent: false
+
+  - label: ":newspaper: Newsfile"
+    branches: "!master !develop !release-*"
+    command:
+      - "python3 -m pip install towncrier"
```

Comment:
I'd recommend sticking these commands in a script (or tox.ini) in the target repo that can be run from here, to reduce the chances of having to PR more changes against the pipeline.

Reply:
This is done in https://github.com/matrix-org/matrix-appservice-irc/pull/990/files